src/components/Article/Article.css

```
@@ -1 +1,30 @@
-/* Put article styles here */
+.article {
+  margin: 5em auto 0;
+  padding: 1em;
+  font-size: 140%;
+  max-width: 800px;
+  background: white;
```

```suggestion
  background-color: white;
```
Better to specify, if you are setting only color